packages/frontend-api/src/Model/Mutation/Customer/User/CustomerUserMutation.php

```diff
@@ -86,9 +86,12 @@ public function __construct(
 
     /**
      * @param \Overblog\GraphQLBundle\Definition\Argument $argument
+     * @param \Overblog\GraphQLBundle\Validator\InputValidator $validator
+     * @return \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser
      */
-    public function changePassword(Argument $argument)
+    public function changePassword(Argument $argument, InputValidator $validator): CustomerUser
     {
+        $validator->validate();
```

Comment:
Is it really necessaary here? I thought it is validated by GQL bundle by the definitions.

Reply:
This is how you run that validation :)